### PR TITLE
fix broken references in migration guide

### PR DESCRIPTION
**Pull request by aboedo**
fixed a couple of broken references in the v4 api migration guide

### CODE REVIEW

File: Documentation.docc/V4_API_Migration_guide.md [hunk 1]
```diff
@@ -277,10 +277,10 @@ if let error = error as? RevenueCat.ErrorCode {
 
 ## New APIs
 - All applicable methods have an [async/await alternative](https://docs.swift.org/swift-book/LanguageGuide/Concurrency.html).
-- `showManageSubcriptions(completion:)`: Use this method to show the subscription management for the current user. Depending on where they made the purchase and their OS version, this might take them to the `managementURL`, or open the iOS Subscription Management page.
```
Comment (aboedo) [inline Documentation.docc/V4_API_Migration_guide.md:280]: this was missing an "s" in `subscription`

File: Documentation.docc/V4_API_Migration_guide.md [hunk 2]
```diff
@@ -277,10 +277,10 @@ if let error = error as? RevenueCat.ErrorCode {
...
-- `showManageSubcriptions(completion:)`: Use this method to show the subscription management for the current user. Depending on where they made the purchase and their OS version, this might take them to the `managementURL`, or open the iOS Subscription Management page. 
-- `beginRefundRequestForCurrentEntitlement`: Use this method to begin a refund request for the purchase that granted the current entitlement.
-- `beginRefundRequest(forProduct:)`: Use this method to begin a refund request for a purchase, specifying the product identifier.
-- `beginRefundRequest(forEntitlement:)`: Use this method to begin a refund request for a purchase, specifying the entitlement identifier.
+- ``Purchases/showManageSubscriptions(completion:)``: Use this method to show the subscription management for the current user. Depending on where they made the purchase and their OS version, this might take them to the `managementURL`, or open the iOS Subscription Management page. 
+- ``Purchases/beginRefundRequestForActiveEntitlement()``: Use this method to begin a refund request for the purchase that granted the current entitlement.
```
Comment (aboedo) [inline Documentation.docc/V4_API_Migration_guide.md:281]: the method name for this one was wrong

File: Documentation.docc/V4_API_Migration_guide.md [hunk 3]
```diff
@@ -277,10 +277,10 @@ if let error = error as? RevenueCat.ErrorCode {
...
+- ``Purchases/beginRefundRequest(forProduct:)``: Use this method to begin a refund request for a purchase, specifying the product identifier.
+- ``Purchases/beginRefundRequest(forEntitlement:)``: Use this method to begin a refund request for a purchase, specifying the entitlement identifier.
```
Comment (aboedo) [inline Documentation.docc/V4_API_Migration_guide.md:283]: the double backticks ensure we won't mess these up again

**Comment by aboedo** [comment]
swiftlint will continue to fail until #1411 is merged, but this PR doesn't change actual code so it should be safe

**Review by NachoSoto — APPROVED** [review]
Nice!

**Pull request merged**